docs/source/developing/ci.adoc

```diff
@@ -10,21 +10,51 @@ toc::[]
 [[ci-overview]]
 == Overview
 
-The following sections describes the various CI setups Minishift utilizes.
+Minishift uses several CI build servers such as Centos, AppVeyor, Circle and Travis. These servers build incoming pull requests, pass it through various test parameters and provide the status of the builds to ensure continuous integration and deployment. Except for Travis CI, they also archive build artifacts of successful builds.
```

Comment:
"ensure continuous integration and deployment" - not sure about this. We for sure don't do "continuous deployment".